doc/src/manual/performance-tips.md

```diff
@@ -599,8 +546,7 @@ optimize the body of the loop. There are several possible fixes:
 
   * Initialize `x` with `x = 1.0`
   * Declare the type of `x`: `x::Float64 = 1`
-  * Use an explicit conversion: `x = oneunit(T)`
```

Comment:
In `oneunit(T)`, what is `T` even?

Reply:
Here, it seems like `oneunit(Float64)` would parallel the other examples.